zerver/lib/actions.py

```diff
@@ -3349,7 +3349,7 @@ def do_change_icon_source(realm: Realm, icon_source: str, log: bool=True) -> Non
                               icon_url=realm_icon_url(realm))),
                active_user_ids(realm.id))
 
-def do_change_logo_source(realm: Realm, logo_source: str, night: bool) -> None:
+def do_change_logo_source(realm: Realm, logo_source: str, night: bool, acting_user: UserProfile) -> None:
```

Comment:
Same comment on passing by kwargs; I'd even consider making these functions defined as:
```
def do_change_logo_source(realm: Realm, logo_source: str, night: bool, *, acting_user: UserProfile) -> None:
```

(note the `*`), so that one is required to pass the argument as a kwarg.

Reply:
@timabbott Should we use this style in every function that records RealmAuditLog?

Reply:
At least of the ones in `actions.py` that call `send_event`, yeah.  (No need to migrate the existing ones in this PR, but we'll want to do that eventually).